src/components/story/sections/StoryListSection.tsx

```
@@ -38,13 +38,21 @@ function StoryListSection({
-              title={item.title}
-              content={item.content}
-            />
-          </Link>
+          <div
+            className={css({
+              display: 'flex',
+              flexDirection: 'column',
+              mb: 8,
+            })}
+          >
+            <Link href={`/story/${item.id}`} key={item.id}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)